interop-tests/Dockerfile.native

```diff
@@ -15,7 +14,7 @@ RUN cargo chef cook --release --package interop-tests --bin native_ping --recipe
 COPY . .
 RUN cargo build --release --package interop-tests --bin native_ping
 
-FROM gcr.io/distroless/cc
```

Comment:
What is wrong with `distroless`? Is it not available for ARM? :(

Reply:
I get errors like these:

```
      "rust-v0_52_x_nim-v1_0__tcp__noise__yamux_-dialer-1    | testplan: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by testplan)\n" +
      "rust-v0_52_x_nim-v1_0__tcp__noise__yamux_-dialer-1    | testplan: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by testplan)\n" +
      "rust-v0_52_x_nim-v1_0__tcp__noise__yamux_-dialer-1    | testplan: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by testplan)\n" +
```

Reply:
This is fixed on master where we build them statically linked.